src/components/gui/gui.jsx

```diff
@@ -74,6 +76,8 @@ GUIComponent.propTypes = {
 };
 GUIComponent.defaultProps = {
     basePath: '/',
-    vm: new VM()
+    vm: new VM(),
+    storage: new Storage()
 };
+GUIComponent.defaultProps.vm.attachStorage(GUIComponent.defaultProps.storage);
```

Comment:
I was thinking more like this, as to keep the interface to the GUI simple, since then if you want to use a different storage, you can just pass in your own VM with your storage (and whatever else) attached:
```javascript
const defaultVm = new VM();
defaultVm.attachStorage(new Storage());
GUIComponent.defaultProps = {
    basePath: '/',
    vm: defaultVm,
};
```